fmriprep/workflows/bold/t2s.py

```diff
@@ -98,7 +98,11 @@ def init_bold_t2s_wf(echo_times, mem_gb, omp_nthreads, name='bold_t2s_wf'):
 
     LOGGER.log(25, 'Generating T2* map and optimally combined ME-EPI time series.')
 
-    t2smap_node = pe.Node(T2SMap(echo_times=list(echo_times)), name='t2smap_node')
+    t2smap_node = pe.Node(
+        T2SMap(echo_times=list(echo_times)),
+        name='t2smap_node',
+        mem_gb=2 * mem_gb * len(echo_times),
```

Comment:
Based on @tsalo's measurements, this might be a better estimate.

```suggestion
        mem_gb=2.5 * mem_gb * len(echo_times),
```